src/redux/reducers/componentConfig/index.js

```diff
@@ -36,6 +39,9 @@ const plotsReducer = (state = initialState, action) => {
       return plotDataError(state, action);
     case TRAJECTORY_NODES_SELECTION_UPDATED:
       return trajectoryNodesUpdated(state, action);
+    case CELL_CLASS_DELETE: {
```

Comment:
How will `CELL_CLASS_DELETE` be dispatched? At the moment in the codebase I can't find it dispatched from anywhere.

Reply:
it is already being dispatched, I just added a new reducer for it

Reply:
Thanks, I found it. How did it work before without the reducer?

Reply:
Nothing happened

Reply:
I more meant how did the actual cell class delete get executed before that action, but based on your answer probably it simply was never actually deleted?